lib/teiserver/autohost/tachyon_handler.ex

```diff
@@ -173,7 +176,10 @@ defmodule Teiserver.Autohost.TachyonHandler do
        {:ok,
         %{
           ips: data["ips"],
-          port: data["port"]
+          port: data["port"],
+          engine: %{"version" => data["engine"]},
+          game: %{"springName" => data["game"]},
+          map: %{"springName" => data["map"]}
```

Comment:
prefer using atoms for known keys, they are preallocated and nicer to use. When converted to json, they are automatically converted to string at that point.

Reply:
Done